<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scmSkyCultureDialog</class>
 <widget class="QWidget" name="scmSkyCultureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>516</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000000</width>
    <height>999999</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBar" name="titleBar">
     <property name="title" stdset="0">
      <string>Sky Culture Maker</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="Content">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabs">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>800</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <widget class="QWidget" name="overviewTab">
         <attribute name="title">
          <string>Overview</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="currentSCNameLbl">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Name of the Sky Culture</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="skyCultureNameTE">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentSCIdLbl">
            <property name="text">
             <string>ID of the Sky Culture</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="skyCultureIdTE">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="constellationsLbl">
            <property name="text">
             <string>Constellations:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="constellationsList"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="SaveSkyCultureBtn">
              <property name="text">
               <string>Save Sky Culture</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="AddConstellationBtn">
              <property name="text">
               <string>Add Constellation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="RemoveConstellationBtn">
              <property name="text">
               <string>Remove Constellation</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="infoBoxLbl">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="commonNamesTab">
         <attribute name="title">
          <string>Common Names</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="boundariesTab">
         <attribute name="title">
          <string>Boundaries</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="licenseTab">
         <attribute name="title">
          <string>License</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QFrame</extends>
   <header location="global">Dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/SkyCultureMaker.qrc"/>
 </resources>
 <connections/>
</ui>
